packages/components/src/toggle-group-control/test/index.tsx

```diff
@@ -208,56 +224,77 @@ describe.each( [
 			expect( jackOption ).not.toBeChecked();
 			expect( rigasOption ).toBeChecked();
 
-			await press.ArrowRight();
-
-			expect( rigasOption ).not.toBeChecked();
-			expect( jackOption ).toBeChecked();
```

Comment:
@ciampo Let me know if this second option change was an intentional part of this test. I'd like to remove it to make it a more minimal test for the reset, and because this second maneuver was what was making this test pass with the previous logic.